### ISSUE TITLE
Error on statick --quickstart: invalid syntax ?

### ISSUE THREAD

**Issue by vindarel** [thanethomson/statik#28]
Hi !
I installed statik in a venv with your recommendation (using pip3),
on `statick --quickstart` I get this:

```
$ statik --quickstart      
Traceback (most recent call last):
  File "/home/vince/bacasable/statiktest/bin/statik", line 7, in <module>
    from statik.cmdline import main
  File "/home/vince/bacasable/statiktest/lib/python3.4/site-packages/statik/cmdline.py", line 7, in <module>
    from statik.generator import generate
  File "/home/vince/bacasable/statiktest/lib/python3.4/site-packages/statik/generator.py", line 3, in <module>
    from statik.project import StatikProject
  File "/home/vince/bacasable/statiktest/lib/python3.4/site-packages/statik/project.py", line 12, in <module>
    from statik.database import StatikDatabase
  File "/home/vince/bacasable/statiktest/lib/python3.4/site-packages/statik/database.py", line 398
    **kwargs,
            ^
SyntaxError: invalid syntax
```

Is there a trailing coma [here](https://github.com/thanethomson/statik/blob/master/statik/database.py#L398) or did I unlearn python during summer ?

thx and keep up !

**Comment by thanethomson** [comment]
Hi @vindarel! I think I've seen that kind of error before, and it's something to do with Python 3.4 specifically. I built Statik using Python 3.5 it and seems to work fine - please try with 3.5 and let me know if the error persists?

**Comment by thanethomson** [comment]
@vindarel - please let me know if you've tested with Python 3.5?

**Comment by vindarel** [comment]
hey not yet I wasnt home !

**Comment by vindarel** [comment]
I didn't try with python 3.5, but I simply removed this trailing coma and it's working fine, as expected. 

I found the wiki walkthrough very nice (but the example may not be working, have to check).

**Comment by thanethomson** [comment]
Cool, I've changed this and have released a new version (`v0.6.7`).
